controllers/users.js

```diff
@@ -142,9 +142,11 @@ const getUsers = async (req, res) => {
         try {
           result = await dataAccess.retrieveUsers({ discordId });
           user = result.user;
-
           if (!result.userExists) {
-            return res.boom.notFound("User doesn't exist");
+            return res.json({
+              message: "No user found",
```

Comment:
```suggestion
              message: "user not found",
```

Reply:
**No user found** v/s **user not found** are same things.

Reply:
but user not found is more appropriate from my point of 
ask @iamitprakash for confirmation

Reply:
+1